src/main/java/school/lemon/changerequest/java/introduction/hw1/Task6.java

```diff
@@ -1,11 +1,19 @@
 package school.lemon.changerequest.java.introduction.hw1;
 
 public class Task6 {
+    private static int factorial(int n) {
+        if (n == 0) return 1;
+        return n * factorial(n - 1);
+    } //мастерский метод для нахождения факториала
+
     public static double calculateS(double x) {
-        return 0d;
+
+        return (1+x+ Math.pow(x, 2)/factorial(2)+Math.pow(x, 3)/factorial(3)+Math.pow(x, 4)/factorial(4));
```

Comment:
And what if you'll need to calculate 10 more terms ?
For example: 
x^2/2!+x^3/3!+...+x^10/10!

Could you refactor your code to be more generic ?